resources/opensearchservice-packages.go

```diff
@@ -0,0 +1,62 @@
+package resources
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/opensearchservice"
+	"github.com/rebuy-de/aws-nuke/v2/pkg/types"
+)
+
+type OSPackage struct {
+	svc         *opensearchservice.OpenSearchService
+	packageID   *string
+	packageName *string
+	createdTime *time.Time
+}
+
+func init() {
+	register("OSPackage", ListOSPackages)
+}
+
+func ListOSPackages(sess *session.Session) ([]Resource, error) {
+	svc := opensearchservice.New(sess)
+
+	listResp, err := svc.DescribePackages(&opensearchservice.DescribePackagesInput{})
+	if err != nil {
+		return nil, err
+	}
+
+	resources := make([]Resource, 0)
+
+	for _, pkg := range listResp.PackageDetailsList {
+		resources = append(resources, &OSPackage{
+			svc:         svc,
+			packageID:   pkg.PackageID,
+			packageName: pkg.PackageName,
+			createdTime: pkg.CreatedAt,
+		})
+	}
+
+	return resources, nil
```

Comment:
Sorry, another thing I just noticed - both resources need paging as well, just like you did in #1044. Could you do that?

Reply:
@der-eismann No worries, updated with pagination and noted for my next PR's!